src/core/network/client.ts

```diff
@@ -4,10 +4,10 @@ import * as fs from 'fs'
 import * as http from 'http'
 import * as https from 'https'
 import * as path from 'path'
-import { compactOptions } from '../helpers/compact'
 import fetch, { RequestInfo, RequestInit } from 'node-fetch'
 import { hasProp, hasPropType } from '../helpers/check'
 import { Opts, Telegram } from '../../telegram-types'
+import { compactOptions } from '../helpers/compact'
```

Comment:
Doesn't moving it add a warning?

Reply:
Quite the contrary, it removes a warning.

`Expected 'multiple' syntax before 'single' syntax. eslint(sort-imports)`